ui_testing/elements.py

```diff
@@ -558,6 +558,8 @@
                        'value': '//*[@id="custom-accordion-panel"]/div/a/i'},
         'archive': {'method': 'link_text',
                     'value': 'Archive'},
+        'configurations': {'method': 'link_text',
```

Comment:
it's already defined in general so no need to re-define it
https://github.com/modeso-open-source/1lims-automation/blob/ba7e0d580646ba7cceec19fc3da33c00f1fd1b0f/ui_testing/elements.py#L111